editor/editor_settings.cpp

```diff
@@ -1497,10 +1497,35 @@ void EditorSettings::set_favorites(const Vector<String> &p_favorites) {
 	}
 }
 
+void EditorSettings::set_favorite_properties(const HashMap<StringName, PackedStringArray> p_favorite_properties) {
+	favorite_properties = p_favorite_properties;
+	String favorite_properties_file;
+	if (Engine::get_singleton()->is_project_manager_hint()) {
+		favorite_properties_file = EditorPaths::get_singleton()->get_config_dir().path_join("favorite_properties");
+	} else {
+		favorite_properties_file = EditorPaths::get_singleton()->get_project_settings_dir().path_join("favorite_properties");
+	}
+
+	Ref<ConfigFile> cf;
+	cf.instantiate();
+	for (HashMap<StringName, PackedStringArray>::ConstIterator I = p_favorite_properties.begin(); I;) {
```

Comment:
Why not use KeyValue range iteration?

Reply:
Been a little while since I used those iterators, so the reason is: i forgor :skull:

Reply:
You used them in #97257 though 🤔
Also remember about KV.

Reply:
> You used them in https://github.com/godotengine/godot/pull/97257 though 🤔

My goldfish memory combined with how busy my week has been makes anything past a day or two feel like a millennia.

> Also remember about KV.

Some questions about the `KV` stuff:
- Is it `KV` or `kv`? `kv` seems to be used the most.
- What's the convention when there's a `KeyValue` iteration inside another?

Reply:
- KV. It applies to new code, there are no plans to "fix" the already existing loops.
- No idea. IMO "KV2" would be fine, but maybe ask on the chat, or just use whatever.